src/workerd/api/wpt/dom/abort-test.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+import { type TestRunnerConfig } from 'wpt:harness';
+
+export default {
+  'abort-signal-any.any.js': {
+    includeFile: 'abort-signal-any-tests.js',
+    comment:
+      '(1, 2) Target should be set to signal. (3) Should be investigated.',
+    expectedFailures: [
+      'AbortSignal.any() follows a single signal (using AbortController)',
+      'AbortSignal.any() follows multiple signals (using AbortController)',
+      'Abort events for AbortSignal.any() signals fire in the right order (using AbortController)',
+    ],
+  },
+  'event.any.js': {
+    comment: 'Target should be set to signal',
+    expectedFailures: ['the abort event should have the right properties'],
```

Comment:
The event has `target = undefined` and `currentTarget = signal`. I'm not sure if I'm doing something wrong in the harness or if we have a bug.

Reply:
@jasnell I'm marking this as resolved for now to prepare for PR merge, but here were the failing cases we talked about last time.